config.json.EXAMPLE

```diff
@@ -1,4 +1,8 @@
 {
+    "secretManager": {
+        "enable": false,
+        "secretID": ""
```

Comment:
This stors the secretID in the mm config file. Can't that be used to fetch the privateKey as well?

Reply:
Actually no, you will also need credentials for fetching the data.